### PR TITLE
Adds CK::chain helper function to chain vectors

### PR DESCRIPTION
**Pull request by MDSilber**

### CODE REVIEW

File: ComponentKit/Utilities/CKVectorHelpers.h [hunk 1]
```diff
@@ -0,0 +1,29 @@
+//
+//  CKVectorHelpers.h
```
Comment (adamjernst) [inline ComponentKit/Utilities/CKVectorHelpers.h]: let's just put this in ComponentUtilities.h, just like CK::map

File: ComponentKitTests/CKVectorHelperTests.mm [hunk 2]
```diff
@@ -0,0 +1,75 @@
+//
+//  CKVectorHelperTests.m
+//  ComponentKit
+//
+//  Created by Mason Silber on 6/7/16.
```
Comment (adamjernst) [inline ComponentKitTests/CKVectorHelperTests.mm]: fix copyright header to match other files

**Comment by adamjernst** [comment]
Awesome stuff!

File: ComponentKit/Utilities/CKVectorHelpers.h [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+//
+//  Created by Mason Silber on 6/7/16.
+//
+//
+
+#import <vector>
+
+namespace CK {
+  template <class T> typename
+  std::vector<T> chain(std::vector<T> a, std::vector<T> b) {
```
Comment (eczarny) [inline ComponentKit/Utilities/CKVectorHelpers.h]: Some documentation would also be helpful.

**Comment by adamjernst** [comment]
See comment on issue... I think we either need a new name or different behavior.

**Pull request merged**